### ISSUE TITLE
React: Language selector - dropdown not identified on JAWS shortcut keys

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4330]
## Detailed description
On JAWS FF

Issue: 
This dropdown is not identified by JAWS quick navigation keys but functions correctly via tab focus. Since it is not blocking functionality and still accessible, @Praveen-sr and Ram have agreed it can wait until the next release and lowering the severity. This only pertains to desktop breakpoints and browsers, since mobile device screen readers correctly identify the selector.

Excepted:
Screen reader should be able to Identify the dropdown with JAWS quick navigation keys.

**Comment by kennylam** [comment]
@RobertaJHahn @Praveen-sr a PR has been opened for this issue https://github.com/carbon-design-system/carbon-for-ibm-dotcom/pull/4335

**Comment by Praveen-sr** [comment]
@kennylam Re tested on the new PR link, above issue is fixed, Working fine now.

But new issue is found,
Issue: There is no label for this dropdown

Expected: Provide label for the dropdown using html label tag or aria label
